### PR TITLE
Iteration 1

### PR DESCRIPTION
**Pull request by servamdak**
Created basic configuration of Emarket project from Iteration #01 task.

### CODE REVIEW

File: emarket-traning-proj.iml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (taras-yaroshchuk) [inline emarket-traning-proj.iml]: Please, add `emarket-traning-proj.iml` to .gitignore

File: target/classes/application.properties [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#spring.datasource.url=
```
Comment (taras-yaroshchuk) [inline target/classes/application.properties]: Please add /target dir to .gitignore.
"target" dir is generated by maven when you "mvn clean install", we don't need this in git repository

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#spring.datasource.url=
```
Comment (taras-yaroshchuk) [inline src/main/resources/application.properties:1]: Good 👍 
later in this property file you will be able to add your app.name, port (8080 by default), etc.

**Review by taras-yaroshchuk — CHANGES_REQUESTED** [review]
Look good in general, but there are few unnecessary files added to git. Please, improve you .gitignore

File: emarket-traning-proj.iml (hunk 1 above)
Comment (servamdak) [reply emarket-traning-proj.iml]: Added "emarket-traning-proj.iml" to .gitignore.

File: target/classes/application.properties (hunk 2 above)
Comment (servamdak) [reply target/classes/application.properties]: Added "target" dir to .gitignore.

File: src/main/resources/application.properties (hunk 3 above)
Comment (servamdak) [reply src/main/resources/application.properties:1]: Thank's.

**Review by taras-yaroshchuk — APPROVED** [review]
👍

**Comment by taras-yaroshchuk** [comment]
Please, use "Squash and merge" while merging

**Pull request merged**